recipes/read-roi/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "read-roi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hadim/read_roi
+  git_rev: {{ version }}
```

Comment:
Why that since there is the `git_rev` ? What is the difference since the integrity of the source should be the same.